src/content/developers/docs/consensus-mechanisms/pos/index.md

```diff
@@ -3,85 +3,64 @@ title: Proof-of-stake (PoS)
+Proof-of-stake is a type of [consensus mechanism](/developers/docs/consensus-mechanisms/) used by blockchains to achieve distributed consensus. Whereas in PoW miners prove they have capital at risk by expending energy, in PoS validators explicitly stake capital in the form of ether into a smart contract on Ethereum. This staked ether then acts as collateral that can be destroyed if the validator behaves dishonestly or lazily. The validator is then responsible for checking that new blocks propagated over the network are valid and occasionally creating and propagating new blocks themselves.
 
 Proof-of-stake comes with a number of improvements to the proof-of-work system:
 
-- better energy efficiency – you don't need to use lots of energy mining blocks
-- lower barriers to entry, reduced hardware requirements – you don't need elite hardware to stand a chance of creating new blocks
-- stronger immunity to centralization – proof-of-stake should lead to more nodes in the network
-- stronger support for [shard chains](/upgrades/shard-chains/) – a key upgrade in scaling the Ethereum network
-
-## Proof-of-stake, staking, and validators {#pos-staking-validators}
-
-Proof-of-stake is the underlying mechanism that activates validators upon receipt of enough stake. For Ethereum, users will need to stake 32 ETH to become a validator. Validators are chosen at random to create blocks and are responsible for checking and confirming blocks they don't create. A user's stake is also used as a way to incentivise good validator behavior. For example, a user can lose a portion of their stake for things like going offline (failing to validate) or their entire stake for deliberate collusion.
-
-## How does Ethereum's proof-of-stake work? {#how-does-pos-work}
-
-Unlike proof-of-work, validators don't need to use significant amounts of computational power because they're selected at random and aren't competing. They don't need to mine blocks; they just need to create blocks when chosen and validate proposed blocks when they're not. This validation is known as attesting. You can think of attesting as saying "this block looks good to me." Validators get rewards for proposing new blocks and for attesting to ones they've seen.
-
-If you attest to malicious blocks, you lose your stake.
-
-### The beacon chain {#the-beacon-chain}
-
-When Ethereum replaces proof-of-work with proof-of-stake, there will be the added complexity of [shard chains](/upgrades/shard-chains/). These are separate blockchains that will need validators to process transactions and create new blocks. The plan is to have 64 shard chains, with each having a shared understanding of the state of the network. As a result, extra coordination is necessary and will be done by [the beacon chain](/upgrades/beacon-chain/).
+- better energy efficiency – there is no need to use lots of energy on PoW computations
```

Comment:
Could we use proof-of-work / proof-of-stake over the acronym [as per the style guide](https://ethereum.org/en/contributing/style-guide/#proof-of)

Reply:
done

Reply:
Hmmm I had to fix a bunch of these and realised that this maybe isn't actually clear:

We stick to spelling out full terms (proof-of-stake) over acronyms (PoS) to avoid the potential cognitive overhead it causes.

See [this section of the Google Technical Writing Guide](https://developers.google.com/tech-writing/one/words?hl=en#use_the_acronym_or_the_full_term) for the reasoning we used to inform this choice.